docs/resources/built-in-decorators.md

```diff
@@ -65,3 +65,21 @@ lint:
       tagNames:
         pet: ./my-custom-description.md
 ```
+
+### remove-x-internal
+
+Remove all nodes that have a specific `internalPropertyName` property (`x-internal` by default).
+
+Remove additional remnants from components by also using the `remove-unused-components` decorator.
+
+```yaml
+lint:
+  decorators:
+    remove-x-internal:
+      internalPropertyName: 'removeit'
+```
+
+### remove-unused-components
+
+Removes unused components.
+Also can be used as a CLI argument `--remove-unused-components` to the `bundle` command.
```

Comment:
I would not expose it as a decorator as it has to be run last which is a potential place for confusing/unexpected behaviors.

Reply:
done